docs/key-issues/gpus.md

```diff
@@ -180,3 +204,85 @@ Potential solutions on the PyPI side include:
+runtime shared library than the one they were compiled against. In this scenario,
+users are responsible for having the right CUDA runtime library installed.  As
+a result, the official CUDA recommendation is to statically link the CUDA
+runtime (see
+[here](https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#recommendations-for-building-a-minor-version-compatible-library)
+and
+[here](https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#distributing-the-cuda-runtime-and-libraries)
+for more information).
+
+CUDA drivers have always been backwards compatible. Any code that runs when
+some driver version X is installed will always work correctly when run with
+some newer driver version Y>X. As briefly discussed above, as of CUDA 11.0
+CUDA also promises [minor version
+compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html#minor-version-compatibility)
+(MVC). This compatibility guarantees that CUDA code compiled using a certain
+version of the CUDA toolkit will be compatible with any driver version within
+the same major release. This behavior is useful because it is often easier for
+users to upgrade their CUDA runtime than it is to upgrade the driver,
+especially on shared machines. For instance, the CUDA toolkit may be installed
+using conda, while the driver library cannot be. An example of leveraging MVC
+would be compiling code against the CUDA 11.5 runtime library and then running
+it on a system with a CUDA 11.2 driver installed. However there are some
+caveats with MVC:
+
+- If CUDA source code uses any features that were introduced in a later driver
+  version than the installed version, it will still fail to run. However, it
+  will be a runtime failure in the form of a `cudaErrorCallRequiresNewerDriver`
+  CUDA error, rather than a linker error or some similarly opaque issue. One
+  solution to this problem is for libraries to use runtime checks of the CUDA
+  version (using e.g. `cudaDriverGetVersion`) to only use supported features on
+  the installed driver.
+- NVRTC did not start supporting MVC until CUDA 11.3. Therefore, code that uses
+  NVRTC for JIT compilation must have been compiled with a CUDA version >= 11.3.
+- MVC only applies to SASS binary code, not
+  [PTX](https://docs.nvidia.com/cuda/parallel-thread-execution/). PTX is an
+  instruction set that the CUDA driver library can JIT-compile to machine
+  instructions. The standard [CUDA compilation
+  pipeline](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#the-cuda-compilation-trajectory)
+  includes the translation of CUDA source code into PTX. In addition, to
+  achieve the best performance, developers may write PTX code directly or use
+  code generation tools that produce PTX because it offers avenues for
+  optimization that are either unavailable or very difficult to access with
+  CUDA C/C++.  However, since MVC does not support PTX code, PTX generated
```

Comment:
While this is true, I think it's really too detailed. We simply wanna mention that some projects choose to generate PTX here, without justifying for them. I suggest the following change:
```suggestion
  includes the translation of CUDA source code into PTX. In addition, some projects
  choose to include PTX code generated either at build time or run time.
  However, since MVC does not cover JIT'ing PTX code, PTX generated
```